### PR TITLE
Adds an Expeditionary Channel and Intercoms & Gives the Interrogation Channel a Name

### PR DESCRIPTION
**Pull request by 1Sparky1**
Adds Expeditionary intercomms to both the Spark and Intrepid, able to be tuned into on shortwaves. Both Interrogation and Expeditionary comm channels are now labelled in chat, with security and research colours respectively. 

(Yes I know the numbering is wrong)

### CODE REVIEW

**Comment by 1Sparky1** [comment]
!review

**Comment by DreamySkrell** [comment]
PRs should have descriptive titles. Feel free to rename the PR if you can think of a better title though.

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]
Edit `chat-dark.scss` and `chat-light.scss` to give this frequency a unique colour.

File: code/game/objects/items/devices/radio/intercom.dm [hunk 1]
```diff
@@ -174,6 +175,43 @@ pixel_x = 8;
...
+/obj/item/device/radio/intercom/expedition/hailing/south
+	PRESET_SOUTH
+
+/obj/item/device/radio/intercom/expedition/hailing/west
+	PRESET_WEST
+
+/obj/item/device/radio/intercom/expedition/hailing/east
+	PRESET_EAST
+
+/obj/item/device/radio/intercom/expedition/hailing/Initialize()
```
Comment (Arrow768) [inline code/game/objects/items/devices/radio/intercom.dm:210]:
```suggestion
/obj/item/device/radio/intercom/hailing/Initialize()
```
Is there anything that makes this radio specific to expeditions?
-> At a quick glance it looks like a normal radio with the hailing channel enabled.

**Review by Arrow768 — CHANGES_REQUESTED** [review]

File: code/game/objects/items/devices/radio/intercom.dm (hunk 1 above)
Comment (1Sparky1) [reply code/game/objects/items/devices/radio/intercom.dm:210]: Expedition intercoms can only be used to tune into expedition and hailing channels, so a physician can't tune it into medical comms or any other non-public channel. The difference between the regular and hailing versions here is that the hailing one starts tuned to hailing so the Intrepid and Spark can hear any hails made to them.

**Review by NonQueueingMatt — APPROVED** [review]

**Review by DreamySkrell — APPROVED** [review]

**Pull request merged**